scylla/src/lib.rs

```diff
@@ -96,6 +96,7 @@
 pub use scylla_cql::frame;
 pub use scylla_cql::macros::{self, *};
 
+pub mod authenticate;
```

Comment:
IMO `authentication` would be a better name for the module. Module names are usually nouns, not verbs. The `frame::response::authenticate` module is called like that most likely because it contains a definition of the `AUTHENTICATE` frame (although it contains definitions of other frames, too, so maybe at some point we should change it).